### PR TITLE
Minor improvements to French translation

### PR DESCRIPTION
**Pull request by alexandreleroux**

### CODE REVIEW

File: KeepingYouAwake/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -32,4 +32,3 @@ Click droit pour le menu";
 
 /* Version */
 "Version" = "Version";
-
```
Comment (newmarcel) [inline KeepingYouAwake/fr.lproj/Localizable.strings]: I personally prefer having a empty line at the end of source and `.strings` files.

File: KeepingYouAwake/fr.lproj/Preferences.strings [hunk 2]
```diff
@@ -72,4 +72,3 @@
 
 /* Class = "NSTextFieldCell"; title = "Automatically opens the app when you start your Mac."; ObjectID = "x6s-XD-lBU"; */
 "x6s-XD-lBU.title" = "Lance KeepingYouAwake au démarrage du Mac.";
-
```
Comment (newmarcel) [inline KeepingYouAwake/fr.lproj/Preferences.strings]: Please keep the empty line at the end of the file.

**Comment by alexandreleroux** [comment]
@newmarcel I can't resolve the conflict myself unless you give me write access to the repository. The warning message I get is: 
```
This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests.
```
Our options, as far as I know, are (A) you resolve the conflict or (B) you grant me write access. Thx :-) -- Alex

**Comment by newmarcel** [comment]
OK, no problem, I will pull your commits directly into `master` and resolve the conflicts 😄

**Comment by newmarcel** [comment]
I've integrated both commits into `master`.

**Pull request closed without merging**